src/main/java/bio/terra/workspace/app/controller/UnauthenticatedApiController.java

```diff
@@ -24,4 +29,16 @@ public UnauthenticatedApiController(WorkspaceManagerStatusService statusService)
     return new ResponseEntity<>(
         currentStatus, currentStatus.getOk() ? HttpStatus.valueOf(200) : HttpStatus.valueOf(500));
   }
+
+  @Override
+  public ResponseEntity<SystemVersion> serviceVersion() {
+    SystemVersion currentVersion =
+        new SystemVersion()
+            .tag(versionConfiguration.getTag())
+            .hash(versionConfiguration.getHash())
+            .github(
+                "https://github.com/DataBiosphere/terra-workspace-manager/commit/"
+                    + versionConfiguration.getHash());
```

Comment:
this `currentVersion` object will not change for the life of the running java service. So, you could create `currentVersion` once at the class level instead of re-creating a new one each time this method runs.